ingestor/src/main/resources/logback.xml

```diff
@@ -0,0 +1,14 @@
+<configuration>
+
+  <appender name="STDOUT" class="ch.qos.logback.core.ConsoleAppender">
+    <!-- encoders are assigned the type
+         ch.qos.logback.classic.encoder.PatternLayoutEncoder by default -->
+    <encoder>
+      <pattern>%d{HH:mm:ss.SSS} [%thread] %-5level %logger{36} - %msg%n</pattern>
+    </encoder>
+  </appender>
+
+  <root level="debug">
+    <appender-ref ref="STDOUT" />
+  </root>
+</configuration>
```

Comment:
It is possible to move the log config into code and then share that between projects as here: 

https://github.com/guardian/grid/blob/master/common-lib/src/main/scala/com/gu/mediaservice/lib/LogConfig.scala

Then include:

https://github.com/guardian/grid/blob/master/thrall/app/Global.scala#L9

^ That example will need to be reworked for Finatra but the idea is sound I think.